packages/loader/container-loader/src/loader.ts

```diff
@@ -433,7 +433,7 @@ export class Loader extends EventEmitter implements ILoader {
         // If set in both query string and headers, use query string
         request.headers[LoaderHeader.version] = parsed.version ?? request.headers[LoaderHeader.version];
 
-        // Version === null means not use any snapshot.
+        // Version === null means use oldest snapshot.
```

Comment:
Can you please clarify what "oldest snapshot" means? I.e. how is it different from not passing LoaderHeader.version?
If it's the same, then I'd argue we should not allow / support null here, there is already a way to load latest - by doing nothing (not passing anything).

Reply:
i believe undefined in latest, null is oldest, and anything else is specific

Reply:
null used to mean load from ops, but that doesn't really exist for detached anymore. it would be nice if there was an op that told us to go load the summary if we haven't already, rather than doing this trickery with storage

Reply:
https://github.com/microsoft/FluidFramework/pull/4259#discussion_r518927846